completions/_quick-lint-js

```diff
@@ -0,0 +1,22 @@
+#compdef _quick-lint-js quick-lint-js
```

Comment:
This is not a comment, though I don't see the difference in working of script even if we redact `#`.

I've two reasons to use it as it is:
1. [zsh docs](http://zsh.sourceforge.net/Doc/Release/Completion-System.html)
> If the number of completion files changes, compinit will recognise this and produce a new dump file. However, if the name of a function or the arguments in the first line of a #compdef function (as described below) change, it is easiest to delete the dump file by hand so that the next time compinit will re-create it.

2. [zsh-users HowTo](https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org#telling-zsh-which-function-to-use-for-completing-a-command)
> The first line of a completion function file can look something like this:
> 
> #compdef foobar
> 
> This tells zsh that the file contains code for completing the foobar command. This is the format that you will use most often for the first line, but you can also use the same file for completing several different functions [..SNIP..]